feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md

```diff
@@ -0,0 +1,30 @@
+# gNMI-??: Telemetry: Fan Tray Type Test
+
+## Summary
+
+Validate the type of fan tray components.
+
+## Procedure
+
+*   For every fan tray component on a device (regardless of whether or not the fans in the fan tray are considered FRU fans):
+
+    *   Validate that the string in /components/component/state/type is equal to either "FAN_TRAY" or "openconfig-platform-types:FAN_TRAY" (for devices that use this prefix).
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test.  OC paths used for test setup are not listed here.
+
+```yaml
+paths:
+  ## State Paths ##
+  /components/component/state/type:
+
+rpcs:
+  gnmi:
+    gNMI.Subscribe:
+```
+
+
+## Minimum DUT platform requirement
```

Comment:
@dplore I am not familiar with DUT platform requirements, but it looks like some other tests leave this as N/A.

Reply:
For this, "MFF" should be used.  (Modular form factor devices). This is expected to be a chassis with removable line cards, fabrics, controller_card and in this case, FAN_TRAY.  Of course, this is not the only type of device that might have a FAN_TRAY, but is the most common which would.   This documentation is a hint to the test runner which class of hardware should be used.

Reply:
Ok thank you!